### PR TITLE
Dave/traffic in only

### PR DESCRIPTION
**Pull request by davidbeesley**

### CODE REVIEW

**Review by sehz — APPROVED** [review]

File: crates/fluvio-protocol/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "fluvio-protocol"
 edition = "2021"
-version = "0.8.3"
```
Comment (sehz) [inline crates/fluvio-protocol/Cargo.toml:4]: why this need to be bump up?

**Comment by sehz** [comment]
Also, make sure existing metrics is not changed, there might be downstream consequence

File: crates/fluvio-protocol/Cargo.toml (hunk 1 above)
Comment (morenol) [reply crates/fluvio-protocol/Cargo.toml:4]: It is due to the changes in the other branch

Comment (davidbeesley) [reply crates/fluvio-protocol/Cargo.toml:4]: Yeah this is going into integration branch. We should double check when integration is moving to master that it's correct.

**Comment by davidbeesley** [comment]
No longer necessary.

**Pull request closed without merging**